src/sys/inotify.rs

```diff
@@ -27,7 +27,7 @@
 //!     let event = instance.read_event()
 //!         .expect("Failed reading inotify events");
 //!     
-//!     println!("wd: {}, mask: {:?}, cookie: {}, name: {:?}",
+//!     println!("wd: {:?}, mask: {:?}, cookie: {}, name: {:?}",
```

Comment:
Since you derive `Debug` for `InotifyEvent`, you can simply do `println!("{:?}", event);` instead of printing each field individually.